### ISSUE TITLE
NOT an issue MAYBE a new feature

### ISSUE THREAD

**Issue by Sekret0** [Jessecar96/SteamDesktopAuthenticator#966]
There is any chance to add an new feature to RENAME the accounts into SDA?

![Screenshot_1](https://github.com/Jessecar96/SteamDesktopAuthenticator/assets/137343422/f0ab79bb-cd45-4708-8313-1a02b2c813b8)

**Comment by dyc3** [comment]
No, but if you have encryption disabled, you can open the maFile in a text editor and edit the `account_name` field.

**Comment by Sekret0** [comment]
> No, but if you have encryption disabled, you can open the maFile in a text editor and edit the `account_name` field.

Nice... I'll try it. Ty in advance

**Comment by Sekret0** [comment]
> No, but if you have encryption disabled, you can open the maFile in a text editor and edit the `account_name` field.

thank you a lot for the tip... Much more easy rename and sort my 500 accounts

**Comment by invisy** [comment]
> > No, but if you have encryption disabled, you can open the maFile in a text editor and edit the `account_name` field.
> 
> thank you a lot for the tip... Much more easy rename and sort my accounts

I`ve opened issues just to find the same enhancement offer
